docs/source/workflows/predictor_evaluation_workflows.rst

```diff
@@ -0,0 +1,264 @@
+
+The predictor we'll evaluate is defined below:
+
+.. code:: python
+
+    from citrine.informatics.predictors import SimpleMLPredictor
+    from citrine.informatics.descriptors import RealDescriptor
+
+    x = RealDescriptor('x', lower_bound=0.0, upper_bound=1.0)
+    y = RealDescriptor('y', lower_bound=0.0, upper_bound=1.0)
+
+    data_source = CSVDataSource(
+        filename, # path to CSV that contains training data for x and y
+        column_definitions={'x': x, 'y': y}
+    )
+
+    predictor = SimpleMLPredictor(
+        name='y predictor',
+        description='predicts y given x',
+        inputs=[y],
+        outputs=[x],
+        latent_variables=[],
+        training_data=[data_source]
+    )
+
+This predictor expects ``x`` as an input and predicts ``y``.
+Training data is provided by a :class:`~citrine.informatics.data_sources.CSVDataSource` that assumes ``filename`` represents the path to a CSV that contains ``x`` and ``y``.
+
+Next, we'll create a cross-validation evaluator for the response ``y`` with 8 folds and 3 trials and request metrics for root-mean square error (:class:`~citrine.informatics.predictor_evaluation_metrics.RMSE`) and predicted vs. actual data (:class:`~citrine.informatics.predictor_evaluation_metrics.PVA`).
+
+.. note::
+    Here, we're performing cross-validation on an output, but latent variables are valid cross-validation responses as well.
+
+.. code:: python
+
+    from citrine.informatics.predictor_evaluator import CrossValidationEvaluator
+    from citrine.informatics.predictor_evaluation_metrics import RMSE, PVA
+
+    evaluator = CrossValidationEvaluator(
+        name='cv',
+        n_folds=8,
+        n_trials=3,
+        responses={'y'},
+        metrics={RMSE(), PVA()}
+    )
+
+Then add the evaluator to a :class:`~citrine.informatics.workflows.predictor_evaluation_workflow.PredictorEvaluationWorkflow`, register it with your project and wait for validation to finish:
+
+.. code:: python
+
+    from citrine.seeding.find_or_create import find_or_create_project
+    from citrine.informatics.workflows import PredictorEvaluationWorkflow
+    from citrine.jobs.waiting import wait_while_validating
+
+    api_key = os.environ.get('CITRINE_API_KEY')
+    client = Citrine(api_key)
+    project = find_or_create_project(client.projects, 'example project')
+
+    workflow = PredictorEvaluationWorkflow(
+        name='workflow that evaluates y',
+        evaluators=[evaluator]
+    )
+
+    workflow = project.predictor_evaluation_workflows.register(workflow)
+    wait_while_validating(project.predictor_evaluation_workflows, workflow)
+
+Trigger the workflow against a predictor to start an execution. Then wait for the results to be ready:
```

Comment:
Should the example code explicitly register the predictor as well?

Reply:
Yes, good point. I'll add that to the code.